### DISCUSSION TITLE
[idom-jupyter] is there anyway to remove the print in the output?

### DISCUSSION THREAD

**Discussion by tony-maynard** [idom-team/idom#656] [Question] [answered]
Using the binder provided by [IDOM](https://hub.gke2.mybinder.org/user/idom-team-idom-jupyter-dl95kqoc/lab/tree/notebooks/introduction.ipynb) the output contains the name of the function with an id of some sort, is there anyway we can not output this?
<img width="461" alt="Screenshot 2022-02-07 at 17 43 52" src="https://user-images.githubusercontent.com/85884885/152843854-3f94732c-1011-461d-b0be-e8cdc83a9aea.png">

**Comment by rmorshea** [comment, marked as answer]
If you do the following, the displayed `repr` will go away:

```python
idom_jupyter.LayoutWidget(App())
```

**Reply by tony-maynard** [reply]
Thanks once again @rmorshea
